src/infra/main/template.yaml

```diff
@@ -40,6 +40,20 @@ Parameters:
     Description: The name of the Core VC Storage stack
     Type: String
     Default: ais-core
+  AlertingSNSHighAlertNotificationTopicARN:
+    Description: "The ARN of the Alerting SNS High Alert Notification topic"
+    Type: AWS::SSM::Parameter::Value<String>
+    Default: "/ais-infra-alerting/SNS/HighAlertNotificationTopic/ARN" #pragma: allowlist secret
+  LambdaDeploymentPreference:
+    Description: "Specifies the configuration to enable gradual Lambda deployments"
```

Comment:
minor point, but I would explain that this setting can be used to set deployment type to all at once or a type of canary deployment